keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,496 @@
+# which prints out the exact tests so that the actual test coverage is clear
+```
+
+In the above invocation, we verify
+
+- Core Conformance
+- HostProcess
+- ActiveDirectory 
+
+functionality for a specific Kubernetes release. The resulting output would be obtained using
+
+```
+sonobuoy status
+```
+
+Which would report a binary result of `Pass` or `Fail`. This tool could thus be used to establish a rigorous standard for portable verification of Kubernetes Windows based products across Vendors for both internal and external diagnostic purposes.
+
+
+##### Implementation details for Sonobuoy
+
+Below is an example of running the Windows Operational Readiness conformance tests with Sonobuoy
+
+```
+sonobuoy plugin install windows-operational-readiness <url>
+sonobuoy run -p windows-operational-readiness
+```
+
+The URL can point to any custom URL. We have published a few examples of Windows e2e tests here: https://github.com/vmware-tanzu/sonobuoy-plugins/tree/master/windows-e2e.  The canonical version of these will be determined by sig-windows once the tooling convergence has evolved to the point that this KEP is ready for general consumption.  See the requirements for GA section for more details on this in the future.
+
+The process will be the same whether running from a URL or local file, e.g. `sonobuoy run -p windows-op-readiness.yaml` or `sonobuoy run -p pluginURL`. We are also testing plugin installation which would make it easier to store, list, and use plugins.
+
+> Question: In this scenario (or option 2), Sonobuoy still has to know which tests to invoke. In the end it is still just invoking e2e.test so neither of this really avoid option 1, it does make it easier though.
+
+### Test Plan
+
+This KEP defines the test plan for such a cluster, but in general, we will update all of our CI jobs to implement it.
+
+### Graduation Criteria
+
+The plan is to introduce the feature as alpha in the v1.24 time frame having the
+initial framework implementation.
+
+At this time, we'll expect that users can either:
+
+1) Leverage a simple process to run specific tests validating a feature, such as...
+```
+	curl coreconformance.1.23.yaml
+	skiptags=`cat coreconformance1.23.yaml` | yq .basicServices.reboot_all_nodes | grep skipTags
+	skiptags=`cat coreconformance1.23.yaml` | yq .basicServices.reboot_all_nodes | grep focusTags
+	e2e.test -skip="$skiptags" -focus="$focustags"
+```
+
+2) Run `sonobuoy` command line tooling, which consumes the above raw daya, to do the same via
+command line flags:
+
+```
+	
+```
+
+#### Alpha -> Beta graduation
+
+- The suite is graduated to beta when the core tests are implemented and running.
+
+#### Beta -> GA graduation
+
+- The suite is graduated to GA when the sub-specification tests are implemented and running.
+
+### Upgrade / Downgrade Strategy
+
+### Version Skew Strategy
+
+N/A
+
+## Production Readiness Review Questionnaire
```

Comment:
This KEP appears to be about building a test suite as opposed to a feature that is enabled and always present inside of a running cluster.  If I've understood that correctly, then the PRR answers aren't required.

To handle this case, explain why PRR does not apply in this section and remove the rest of question.  Here's an example https://github.com/kubernetes/enhancements/pull/3137/files#diff-0ad9f743dbfdf3932906414acf0a79e074af723ad2ffb8e0dab3429f0b664beeR249-R252

Then I'll approve the PRR

Reply:
@deads2k Squashing commits and adding the PRR sections clarification

Reply:
sounds like were all set now